### PR TITLE
checklist added for pull requests

### PR DESCRIPTION
**Pull request by jurassic-marc**

### CODE REVIEW

File: .github/pull_request_checklist.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Description
+Changes made for this branch
+
+## Checklist
+- [ ] All manual tests passing
+- [ ] All unit tests passing 
+- [ ] All UI tests passing
+- [ ] Code reviewed by peer:
+&nbsp;&nbsp;&nbsp;&nbsp; - [ ] Code reviewed by peer
```
Comment (rorysproule1) [inline .github/pull_request_checklist.md]: Code reviewed by peer? yikes

File: .github/pull_request_checklist.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## Description
+Changes made for this branch
+
+## Checklist
+- [ ] All manual tests passing
+- [ ] All unit tests passing 
+- [ ] All UI tests passing
+- [ ] Code reviewed by peer:
+&nbsp;&nbsp;&nbsp;&nbsp; - [ ] No redundant whitespace
```
Comment (matthewdoreilly) [inline .github/pull_request_checklist.md]: Why all the non-breaking spaces here? Remove these.

Comment (jurassic-marc) [reply .github/pull_request_checklist.md]: @matthewdoreilly was just to indicate that those items are to be checked during 'code review by peer'

Comment (matthewdoreilly) [reply .github/pull_request_checklist.md]: Ah ok that's fine. How does the markdown look on preview though? Cause it's formatted strange here, https://github.com/sapphireduffy/KainosCareerLattice/blob/2926af4180a17d762783f2fa5e7c2b539d99dcf1/.github/pull_request_checklist.md

Comment (matthewdoreilly) [reply .github/pull_request_checklist.md]: You could change the pull request template to the below?

```
## Description
Changes made for this branch

## Checklist
- [ ] All manual tests passing
- [ ] All unit tests passing 
- [ ] All UI tests passing
- [ ] Code reviewed by peer
&nbsp; - No redundant whitespace
&nbsp; - Indentation correct
&nbsp; - Naming conventions followed
&nbsp; - Comments added (where needed)
&nbsp; - No commented out redundant code
&nbsp; - Branch name correct
- [ ] Code reviewed by TA
```

**Pull request merged**